kiwixbuild/dependencies/base.py

```diff
@@ -368,8 +368,9 @@ def set_configure_env(self, env):
 
     def _configure(self, context):
         context.try_skip(self.build_path)
-        command = "{configure_script} {configure_option}"
+        command = "{configure_prefix}{configure_script} {configure_option}"
         command = command.format(
+            configure_prefix=self.buildEnv.configure_prefix,
             configure_script=pj(self.source_path, self.configure_script),
```

Comment:
It seems that the usage of the term "prefix" is somewhat confusing here. I perceive it as a string that has to be prepended to the command name without any space (like turning `g++` into `x86_64-w64-mingw32-g++`, which breaks if we have to handle a path rather than a command: `/build/libfoo/configure` →  `x86_64-w64-mingw32-/build/libfoo/configure`).

Reply:
Yes, I have renamed prefix to wrapper.